src/hm-admin/hm-admin.component.ts

```diff
@@ -1,22 +1,31 @@
 import {Component, Input, OnInit} from '@angular/core';
+import {HTTP_PROVIDERS} from '@angular/http';
 import {ConfigService} from './services/config/config.service';
+import {HydraCore} from './services/hydra/HydraCore.service';
 
 @Component({
   moduleId: module.id,
   selector: 'hm-admin-app',
   templateUrl: 'hm-admin.component.html',
   styleUrls: ['hm-admin.component.css'],
-  providers: [ConfigService]
+  providers: [ConfigService, HydraCore, HTTP_PROVIDERS]
 })
 export class HmAdminComponent implements OnInit {
   @Input('config')
   config: any;
-
-
-  constructor(private configService:ConfigService) {}
+  public hydraCoreDefinition;
```

Comment:
add a blank line here
why repeat hydraCore if you are in a class named hydraCore ?